opm/autodiff/WellDensitySegmented.cpp

```diff
@@ -17,10 +17,10 @@
   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-#include <opm/autodiff/WellDensitySegmented.hpp>
+#include "WellDensitySegmented.hpp"
```

Comment:
Please use the standard include notation with full path, even for headers in the same directory. (This probably applies to several other places as well.)

Reply:
the point of this is that with the quotes, these files can be moved to another directory without having to change them. (we thought about moving them to a `wells/` subdirectory.)

Reply:
While that may be a minor benefit for some cases where the files can be assumed to always be in the same directory (such as here, where a cpp file includes its hpp file) this PR contains many instances of "quote-inclusion" that are not just including the associated header file. Also, it prevents (without changes) moving headers to separate directories which is done by some libraries (although I certainly prefer to have them in the same directory, I am not proposing to change that!).

In this PR it mostly adds unnecessary noise. I was going to ask you to put that commit in its own PR at least, but then this PR is just a single commit...! (Candidate for biggest non-trivial commit in OPM history?)

Reply:
To make it clear: I do not want to mess up the code (further) by introducing this quoting style (I am aware that it already snuck in some places, but I have not had the energy to stamp them out) when the existing style is very clear and good. Should you choose to fight for this change, please use a separate PR, and it would take additional arguments to convince me...!